rake/build/package_st2.rake

```diff
@@ -33,6 +33,7 @@ namespace :package do
             opts.components.each do |component|
               execute :cp, "packages/st2/component.makefile ${GITDIR}/#{component}/Makefile"
             end
+	    execute :bash, '$GITDIR/scripts/populate-sha.sh'
```

Comment:
>Looks like scripts like that could be better to place in current st2-packages repo, rather then st2 repo, since it's part of packaging logic. IMO

The package.meta file will be shipped with the source. I made a call to place it in st2 repo so we could probably run this as part of st2 circle if needed. Once we figure out how to put this file in packages (deb/rpm), we can figure out where to put this script. Do you have any ideas how to achieve that? Basically, I want this fle to be in cat /opt/stackstorm/st2/package.meta after package install.